tests/test_translation.py

```diff
@@ -49,40 +49,60 @@ def setUp(self):
                        "INSTRUME": "SCUBA_test",
                        "DATE-OBS": "2000-01-01T01:00:01.500",
                        "DATE-END": "2000-01-01T02:00:01.500",
+                       "OBSGEO-X": "-5464588.84421314",
+                       "OBSGEO-Y": "-2493000.19137644",
+                       "OBSGEO-Z": "2150653.35350771",
```

Comment:
commenting here about "Basic": Basic what? I assume this is a `BasicTranslatorTestCase`, but why Basic? `TrivialTranslatorTestCase`? Or is it a test of the Translator system itself, in which case just `TranslatorTestCase`?

Reply:
`TranslatorTestCase` is fine. I called it "basic" because it was a simple translator with made up headers.